Automattic/pocket-casts-android
# Fix bookmars not playing when episode is not shown in the episodes list

## Description

Fixes an issue where bookmarks did not play when episodes where filtered out due to a search query.

ecd8f0b122e1412c5d1cff29966ec4e9893bfe3a is cherry-picked from `release/7.58`.

Closes #1816

## Testing Instructions

1. Create a bookmark on an episode
2. Search for something on the Episodes tab of that podcast, so that the episode with a bookmark doesn't get shown.
3. Go to the Bookmarks tab.
4. Play the bookmark.
5. Playback should start.

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/30936061/71b2baa4-17a8-43dc-9653-34efc2dc592d

## Checklist
 
#### I have tested any UI changes...
